### PR TITLE
Correct/Update CI

### LINKED ISSUE #105: PyPI Readme
Also we have to fix the readme on pypi...

_Originally posted by @PicoCentauri in https://github.com/MDAnalysis/mdacli/issues/103#issuecomment-1435915662_

This likely happened in #96 when dropping the function to setup the readme in `setup.py`.

### PR DESCRIPTION
**Pull request by joaomcteixeira**
Closes #105, #107

The problem is that `long_description` in `setup.cfg` can only accept two files. Or at least setting two is does not work the same as for `toml` syntax.

To see if it works:

```
tox -e build
```

Go to `dist/` open the `.whl` file and see the `METADATA` file. The README description for PyPI is there.

### CODE REVIEW

**Comment by orbeckst** [comment]
You can also create the test package on test.PyPi.org — we do this for most of our packages. This will allow you to check everything.

File: setup.py [hunk 1]
```diff
@@ -13,8 +13,14 @@
...
 
+import re
+
 # Always prefer setuptools over distutils
 from setuptools import setup
 
 
-setup()
+VERSION = re.search(
+    r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]', open("src/mdacli/__init__.py").read()
```
Comment (joaomcteixeira) [inline setup.py:23]: Since we use `bump2version`, this is not needed. We can directly inject the version in `pyproject.tolm` or anywhere needed.

**Comment by joaomcteixeira** [comment]
@PicoCentauri , I see you want to move to `pyproject.toml`. So we should update `tox` accordingly and others. Do you agree? Just this week I got a PR and have been working on it https://github.com/joaomcteixeira/python-project-skeleton/pull/33

**Comment by PicoCentauri** [comment]
Thanks for the link. I will update our files accordingly.

**Review by PicoCentauri — APPROVED** [review]
I am happy with the new structure. If you @joaomcteixeira also agree we can merge this.

**Pull request merged**